### PR TITLE
[usage] Enable usage view with feature flag

### PR DESCRIPTION
**Pull request by svenefftinge**
## Description
Since we have discoupled "usage" from stripe now, we can allow all users to use the usage view in order to understand how they use Gitpod (and where their hours go). This is also helpful to get a forecast of how many credits I'd need next month in case I want to switch to usage-based pricing.
This change enabled the usage view for everyone (depending on a feature flag, which is currently disabled for everyone in prod).

## Related Issue(s)

Fixes #

## How to test

Login to the preview environment and see usage view on your individual account and all team accounts.

## Release Notes

```release-note
Added a usage view that displays past workspace sessions on individual and team accounts.
```

## Werft options:

- [x] /werft with-preview
- [x] /werft with-payment

### CODE REVIEW

**Comment by roboquat** [comment]
@svenefftinge: Adding the "do-not-merge/release-note-label-needed" label because no release-note block was detected, please follow our [release note process](https://git.k8s.io/community/contributors/guide/release-notes.md) to remove it.

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

File: components/dashboard/src/contexts/FeatureFlagContext.tsx
Comment (easyCZ) [inline components/dashboard/src/contexts/FeatureFlagContext.tsx:45]: What's the reason for making these very explicit (rather than reflecting over the set of flags)? I feel this has the potential to confuse others who are trying to use feature flags - having to define them here.

Comment (easyCZ) [reply components/dashboard/src/contexts/FeatureFlagContext.tsx:45]: Ah never mind, I missed that you just moved the definition (and the problem already exists)

**Review by easyCZ — APPROVED** [review]
LGTM, assuming we've resolved the Q if usage should be generally visible - [thread](https://gitpod.slack.com/archives/C02EN94AEPL/p1663569777990989)

/hold for Q around consensus on this

**Review by jldec — APPROVED** [review]
LGTM
- team usage view behavior does not depend on the feature flag
- personal usage view requires feature flag

**Pull request merged**
